docs/source/markdown/podman-manifest-exists.1.md

```diff
@@ -7,8 +7,9 @@ podman\-manifest\-exists - Check if the given manifest list exists in local stor
 **podman manifest exists** *manifest*
 
 ## DESCRIPTION
-**podman manifest exists** checks if a manifest list exists on local storage. Podman will
-return an exit code of `0` when the manifest is found. A `1` will be returned otherwise.
+**podman manifest exists** checks if a manifest list exists on local storage.
+Podman returns an exit code of `0` when the manifest is found. A `1` is
+returned otherwise.
```

Comment:
We do this all over the place, and I'm not suggesting we change it., but it drives me a little buggy to see:
"We return 0 on success, 1 otherwise.  An exit code of 125 indicates another issue.".  I'd prefer we'd say something like "0 on success, 1 on an anticipated error, and 125 on an unexpected issue" or some such.

Reply:
This call is checking if the manifest exists 0 indicates it exists, 1 it does not 125, means couldn't check.